lib/phases/phase.js

```diff
@@ -514,7 +519,7 @@ Phase.prototype.calculateStats = function() {
             Min: Math.min.apply(Math, values),
             Max: Math.max.apply(Math, values),
             StdDev: stdev,
-            '95% Bound': mean + (1.96 * stdev / Math.sqrt(values.length))
+            '95% Bound': mean + (1.96 * stdev / Math.sqrt(values.length)) // eslint-disable-line
```

Comment:
522:33  warning  Gratuitous parentheses around expression  no-extra-parens

And I didn't really want to remove the outside parenthesis because it makes it much easier to read.